hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/LazyDefiningSpecificMemberPms.java

```diff
@@ -54,13 +56,11 @@ public LazyDefiningSpecificMemberPms() {
     private LazyDefiningSpecificMemberPms(ProcessorSupplier supplier, int partitionArgumentIndex) {
         this.supplier = supplier;
         this.partitionArgIndex = partitionArgumentIndex;
-        this.memberAddress = null;
     }
 
-    private LazyDefiningSpecificMemberPms(ProcessorSupplier supplier, SupplierEx<?> partitionKeyExprSupplier) {
+    private LazyDefiningSpecificMemberPms(ProcessorSupplier supplier, SupplierEx<Expression<?>> partitionExprSupplier) {
         this.supplier = supplier;
-        this.partitionKeyExprSupplier = partitionKeyExprSupplier;
-        this.memberAddress = null;
+        this.partitionKeyExprSupplier = partitionExprSupplier;
```

Comment:
we have only single expression. Will that work with AttributePartitioningStrategy with >1 fields?

Reply:
This processor is not aware of any AttributePartitioningStrategy, it may work even without declared AttributePartitioningStrategy. Even if it has >1 fields, you can combine them by creating a disjunctive predicate, and then wrap it by supplier. But then, most probably, it doesn't have any sense (it may have >1 partition required).
Actually, I am starting to think if we can create it with more options if we need it...

Reply:
to be described in Jet TDD how SQL can use this in case of >1 field in AttributePartitioningStrategy